src/api/firebase.ts

```diff
@@ -294,10 +295,18 @@ export async function updateItem(listPath: string, item: ListItem) {
 	}
 }
 
-export async function deleteItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to delete an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+//delete an item from the list
+export async function deleteItem(listPath: string, item: ListItem) {
+	const itemDocRef = doc(db, listPath, "items", item.id);
+	// Delete the item from the list in Firestore.
+	// Should be a simple try-catch block. Try === deletes the item. Catch === logs the error.
+	// Let's try the deleteDoc from Firestore.
+	try {
+		await deleteDoc(itemDocRef);
+		alert("Item has been successfully deleted!");
```

Comment:
I like the alert to let the user know if an item's successfully deleted! Maybe an enhancement in the future could be to pull the item name so that the alert confirms if "Cheese has been successfully deleted!" but that is a scope creep for the acceptance criteria of this card I suppose.

Reply:
Fantastic suggestion! A nice touch! Simple to implement!
![image](https://github.com/user-attachments/assets/bdb834bb-563c-4032-89cc-612861f9ca51)

Reply:
Yes! We always want to make sure the user knows they are doing the action they're supposed to!